File: Sources/JXPagerView/JXRTLFlowLayout.m
//
//  JXRTLFlowLayout.m
//
//
//  Created by loong on 2024/3/14.
//

#import "JXRTLFlowLayout.h"

@implementation JXRTLFlowLayout


- (BOOL)flipsHorizontallyInOppositeLayoutDirection {
    return [UIView userInterfaceLayoutDirectionForSemanticContentAttribute:UIView.appearance.semanticContentAttribute] == UIUserInterfaceLayoutDirectionRightToLeft;
}

@end

File: JXRTLFlowLayout.swift
//
//  JXRTLFlowLayout.swift
//  JXPagingView
//
//  Created by Jiaxin Pu on 2024/7/10.
//

import UIKit

class JXRTLFlowLayout: UICollectionViewFlowLayout {
    override var flipsHorizontallyInOppositeLayoutDirection: Bool {
        get {
            return UIView.userInterfaceLayoutDirection(for: UIView.appearance().semanticContentAttribute) == .rightToLeft
        }
    }
}
